spec/controllers/api/v1/gc_organisations_controller_spec.rb

```diff
@@ -88,4 +88,30 @@
       expect(parsed_body).to eq(serialized_gc_organisation)
     end
   end
+
+  describe "GET orders for GC organisation" do
```

Comment:
Could you add some tests for permissions ? to make sure that only entitled users are allowed to call the endpoint